docs/source/whatsnew/v0.22.rst

```diff
@@ -0,0 +1,56 @@
+Version 0.22 (August 3, 2023)
+=============================
+
+Cartopy v0.22 is a major step forward in the project's development. The
+requirement to install with local PROJ and GEOS libraries has been
+removed. The previous C PROJ library calls have been replaced by pyproj
+and the C GEOS calls have been replaced by shapely. This means that
+Cartopy can now be installed with a simple ``pip install cartopy``.
+
+
+For a full list of included Pull Requests and closed Issues, please see the
+`0.22 milestone <https://github.com/SciTools/cartopy/milestone/35>`_.
+
+Features
+--------
+
+* Matthias Cuntz updated the OGC reader to handle EPSG projections. (:pull:`2191`)
+
+* Greg Lucas added the ability to build wheels for the project and
+  move towards new Python packaging standards. (:pull:`2197`)
+
+* Elliott Sales de Andrade removed the GEOS dependency and replaced it with shapely
+  and Greg Lucas added some additional speedups in the geometry transforms. (:pull:`2080`)
+
+* Ruth Comer added the ability to use RGB(A) color arrays with pcolormesh and
+  updated the code to work with Matplotlib version 3.8. (:pull:`2166`)
+
+    .. plot::
```

Comment:
Indented too much; it'll cause a blockquote. Should be aligned with the wrapped text's indent:
```suggestion
  .. plot::
```

Reply:
Good catch, thanks!